content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -112,11 +112,18 @@ that updates those files from a remote source, as in the following diagram:
 
 {{< figure src="/images/docs/pod.svg" alt="Pod creation diagram" class="diagram-medium" >}}
 
-Some Pods have {{< glossary_tooltip text="init containers" term_id="init-container" >}} as well as {{< glossary_tooltip text="app containers" term_id="app-container" >}}. Init containers run and complete before the app containers are started.
+Some Pods have {{< glossary_tooltip text="init containers" term_id="init-container" >}} as well as {{< glossary_tooltip text="app containers" term_id="app-container" >}}. By default, init containers run and complete before the app containers are started.
 
 Pods natively provide two kinds of shared resources for their constituent containers:
 [networking](#pod-networking) and [storage](#pod-storage).
 
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Enabling the [SidecarContainers feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+allows specifying a <code>restartPolicy=Always</code> to init containers, making sure they are restarted when they fail.
```

Comment:
I think this needs a bit more about how it keeps the init container running. Trying to avoid confusion regarding users wondering "Don't failed init containers restart already?".